build/params_testground.go

```diff
@@ -133,9 +133,11 @@ var (
 	Devnet      = true
 	ZeroAddress = MustParseAddress("f3yaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaby2smx7a")
 
-	WhitelistedBlock  = cid.Undef
-	BootstrappersFile = ""
-	GenesisFile       = ""
+	WhitelistedBlock                 = cid.Undef
+	BootstrappersFile                = ""
+	GenesisFile                      = ""
+	F3Enabled                        = false
+	F3BootstrapEpoch  abi.ChainEpoch = -30
```

Comment:
Comment on what this means?

Reply:
I believe this is meant to be "disable value". I have changed it to -1